evm_loader/test_cli_emulate.py

```diff
@@ -245,171 +353,293 @@ def test_successful_cli_emulate(self):
+                    "address": '0x' + self.contract.ethereum_id.hex(),
+                    "code_size": None,
+                    "contract": str(self.contract.contract_code_account),
+                    "new": False,
+                    "writable": True,
+                },
+                {
+                    "account": str(self.caller),
+                    "address": '0x' + self.ethereum_caller.hex(),
+                    "code_size": None,
+                    "contract": None,
+                    "new": False,
+                    "writable": True,
+                },
+            ],
+            "exit_status": "succeed",
+            "result": "0000000000000000000000000000000000000000000000000000000000000001"
+        }
+
+        ether_trx_data = create_ether_trx_data_2(self.token, self.token_acc1, self.token_acc2, self.token_acc3,
+                                                 transfer_amount * (10 ** 9), bytes(self.acc.public_key()))
+
+        emulate_result = emulate_external_call(self.ethereum_caller.hex(),
+                                               self.contract.ethereum_id.hex(),
+                                               ether_trx_data.hex())
+        # no changes after the emulation
+        self.assertEqual(self.spl_token.balance(self.token_acc1), balance1 + mint_amount - transfer_amount)
+        self.assertEqual(self.spl_token.balance(self.token_acc2), balance2 + transfer_amount)
+        self.assertEqual(self.spl_token.balance(self.token_acc3), balance3)
+        self.compare_tmpl_and_emulate_result(tmpl_json, emulate_result)
+
+        emulate_result_uniq = []
+        for item in emulate_result['solana_accounts']:
+            if item not in emulate_result_uniq:
+                emulate_result_uniq.append(item)
+
+        solana_accounts = [AccountMeta(pubkey=item['pubkey'],
+                                       is_signer=item['is_signer'],
+                                       is_writable=item['is_writable'])
+                           for item in emulate_result_uniq]
+
+        print('solana_accounts:', solana_accounts)
+
+        result = self.contract.call(self.ethereum_caller, ether_trx_data, solana_accounts)
+        self.assertNotEqual(result, None)  # https://github.com/neonlabsorg/neon-evm/issues/120
+        src_data = result['result']['meta']['innerInstructions'][-1]['instructions'][-1]['data']
+        self.assertEqual(base58.b58decode(src_data)[0], 6)  # 6 means OnReturn
+        self.assertLess(base58.b58decode(src_data)[1], 0xd0)  # less 0xd0 - success
+
+        self.assertEqual(self.spl_token.balance(self.token_acc1), balance1 + mint_amount - 3*transfer_amount/2)
+        self.assertEqual(self.spl_token.balance(self.token_acc2), balance2 + 2*transfer_amount/2)
+        self.assertEqual(self.spl_token.balance(self.token_acc3), balance3 + 1*transfer_amount/2)
+
+    @unittest.skip("a.i.")
+    def test_emulate_external_call_by_rpc(self):
```

Comment:
What do the symbols `a.i.` mean?
What does this test check?

Reply:
a.i. = temporarily
Remove the test.